README.md

```diff
@@ -5,7 +5,8 @@ A simplified DNS server with a RESTful HTTP API to provide a simple way to autom
 
 ## Why?
 
-Many DNS servers do not provide an API to enable automation for the ACME DNS challenges. And those which do, give the keys way too much power to leave them laying around your random boxes, which sadly would be required to have a meaningful way to automate the process.
+Many DNS servers do not provide an API to enable automation for the ACME DNS challenges. Those which do, gives the keys way too much power.
+Leaving them to lay around your random boxes is too often a requirement to have a meaningful process automation.
```

Comment:
s/to lay/laying/  and if the sentece is separated from the previous, giving context with s/them/the keys/